packages/react/popper/src/Popper.tsx

```diff
@@ -175,53 +175,52 @@ const PopperContent = React.forwardRef<PopperContentElement, PopperContentProps>
       altBoundary: hasExplicitBoundaries,
     };
 
-    const { reference, floating, strategy, x, y, placement, middlewareData, update } = useFloating({
-      // default to `fixed` strategy so users don't have to pick and we also avoid focus scroll issues
-      strategy: 'fixed',
-      placement: desiredPlacement,
-      whileElementsMounted: autoUpdate,
-      middleware: [
-        offset({ mainAxis: sideOffset + arrowHeight, alignmentAxis: alignOffset }),
-        avoidCollisions
-          ? shift({
+    const { reference, floating, strategy, x, y, placement, isPositioned, middlewareData, update } =
+      useFloating({
+        // default to `fixed` strategy so users don't have to pick and we also avoid focus scroll issues
+        strategy: 'fixed',
```

Comment:
`isPositioned` lets you wait until the position is ready now and stick to `absolute`, but that's probably a more intensive change for just upgrading

Reply:
@atomiks I agree, I would better leave it as it is for now.

Reply:
> isPositioned lets you wait until the position is ready now and stick to absolute, but that's probably a more intensive change for just upgrading

@atomiks, can you tell me more about this, specifically around the `absolute` strategy? Are you saying `absolute` should work for everything now? If so, why is there still a strategy option? This is something that we know we want to work out at some point because although `fixed` works consistently for us, we know it isn't the most performant when scrolling, resizing, etc.

Reply:
It means that you can prevent scroll jumps using `absolute`, by doing this:

```js
useLayoutEffect(() => {
  isPositioned && element.focus();
}, [isPositioned]);
```

`fixed` is useful in two cases:

- The reference element is in a fixed container (or is fixed itself) and the floating element isn't inside the fixed container. This prevents jitters when scrolling.
- "Breaking" the floating element out of a clipping container in many cases (when a containing block isn't being formed) to keep DOM context in tact. Not relevant if you're portalling.